meta-ostro/conf/distro/include/ostroproject-ci.inc

```diff
@@ -63,6 +122,9 @@ OSTROPROJECT_CI_TEST_EXPORT_TARGETS="${DISTRO}-image"
 # Explicit machine declaration by build job:
 # MACHINE = "$TARGET_MACHINE"
 
+# Build number shared by all jobs across the different machines:
+# BUILD_ID = "<parent-build-timestamp (of Jenkins top job)>" + "-build-" + "<jenkins-top-job-build-number>"
+
```

Comment:
it is not explicitly set variable, it's exported from jenkins. If you're using it (relaying on it) it's better to say that it's expected in certain format, but it's not something that is really needed for bitbake.